packages/fuel-indexer-database/postgres/src/lib.rs

```diff
@@ -554,6 +554,29 @@ pub async fn latest_assets_for_index(
     })
 }
 
+pub async fn last_block_height_for_indexer(
+    conn: &mut PoolConnection<Postgres>,
+    namespace: &str,
+    identifier: &str,
+) -> sqlx::Result<i64> {
+    #[cfg(feature = "metrics")]
+    METRICS
+        .db
+        .postgres
+        .last_block_height_for_indexer_calls
+        .inc();
+
+    let query = format!(
+        "SELECT MAX(height) FROM {}.block",
+        format_args!("fuel_indexer_{}_{}", namespace, identifier)
+    );
```

Comment:
It's definitely not easy to find, but we have an `IndexMetadata` entity and backing database table that is created for each index; in the codegen, a final call is made to save the block height to said table. You can see that call [here](https://github.com/FuelLabs/fuel-indexer/blob/master/packages/fuel-indexer-macros/src/indexer.rs#L639-L640) and an example using the database table [here](https://github.com/FuelLabs/fuel-indexer/blob/master/packages/fuel-indexer-tests/tests/e2e/indexing_postgres.rs#L591-L594).